presto-product-tests/conf/presto/etc/environment-specific-catalogs/singlenode-kerberos-hdfs-impersonation/hive.properties

```diff
@@ -12,13 +12,13 @@ hive.metastore-cache-ttl=0s
 
 hive.metastore.authentication.type=KERBEROS
 hive.metastore.service.principal=hive/hadoop-master@LABS.TERADATA.COM
-hive.metastore.client.principal=hive/hadoop-master@LABS.TERADATA.COM
-hive.metastore.client.keytab=/etc/hive/conf/hive.keytab
+hive.metastore.client.principal=hive/_HOST@LABS.TERADATA.COM
```

Comment:
please use presto-hive-client/_HOSTS or something like that, that makes it clear that the client principal is distinct from the hive service principal